frontend/main.js

```diff
@@ -1,13 +1,74 @@
 const { app, BrowserWindow, dialog, ipcMain } = require("electron/main");
 const path = require("node:path");
-
+const fs = require("fs-extra");
+const { PythonShell } = require("python-shell");
+const constants = require("./constants/constants.js");
```

Comment:
It might be easier to use destructuring here to import like this

```javascript
const { COMMANDS, OPTIONS, FORMAT } = require("./constants.constants.js");
```